src/openrct2-ui/windows/ShortcutKeys.cpp

```diff
@@ -204,6 +204,7 @@ static const ShortcutStringPair ShortcutList[] =
     { SHORTCUT_ADVANCE_TO_NEXT_TICK,                  STR_ADVANCE_TO_NEXT_TICK },
     { SHORTCUT_PAINT_ORIGINAL_TOGGLE,                 STR_SHORTCUT_PAINT_ORIGINAL },
     { SHORTCUT_DEBUG_PAINT_TOGGLE,                    STR_SHORTCUT_DEBUG_PAINT_TOGGLE },
+    { SHORTCUT_TOGGLE_CLEARANCE_CHECKS,               STR_SHORTCUT_TOGGLE_CLEARANCE_CHECKS },
```

Comment:
The shortcut key window is no longer bound to the internal shortcut key order. Please add this new shortcut into one of the other, more appropriate categories instead.